packages/dropdown/src/presenters/OptionPresenter.js

```diff
@@ -87,7 +88,7 @@ export default class OptionPresenter extends Component {
         <span className="hig__dropdown-option__label">{children}</span>
         {selected && (
           <div className="hig__dropdown-option__checkmark">
-            <CheckmarkBlueDark24 />
+            <CheckmarkBlueDark24 color="#087AAD" />
```

Comment:
should this color be coming from `theme-data`?

Reply:
It doesn't seem to be specified in abstract. The color for this icon is normally hard coded in the SVG file, this is just overriding the default icon color.

Reply:
👍